include/librealsense2/dds/dds-utilities.h

```diff
@@ -0,0 +1,53 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <fastrtps/types/TypesBase.h>
+#include <librealsense2/utilities/easylogging/easyloggingpp.h>
+
+// FastDDS API in most cases is build from 2 formats:
```

Comment:
built
But I would say:
// FastDDS, in most case, is using two error conventions:

Reply:
👍